setup.cfg

```diff
@@ -5,6 +5,8 @@ oteapi.download =
   oteapi.https = oteapi.strategies.download.https:HTTPSStrategy
   oteapi.ftp = oteapi.strategies.download.sftp:SFTPStrategy
   oteapi.sftp = oteapi.strategies.download.sftp:SFTPStrategy
+oteapi.mapping =
+  oteapi.mapping = oteapi.strategies.mapping.mapping:MappingStrategy
```

Comment:
```suggestion
  oteapi.triples = oteapi.strategies.mapping.mapping:MappingTriplesStrategy
```

Reply:
I think I would like to keep the original naming. This is simply a basic mapping strategy that adds mapping triples to the session such that they are available for other strategies.